bin/data-push.js

```diff
@@ -90,11 +90,11 @@ Promise.resolve().then(async () => {
       },
       schedule: argv.schedule
     }
-    await datahub.push(dataset, options)
-
+    let res = await datahub.push(dataset, options)
+    let revisionId = res.flow_id.split('/').pop()
     stopSpinner()
     const message = '🙌  your data is published!\n'
-    const url = urljoin(config.get('domain'), datahubConfigs.owner, dataset.descriptor.name)
+    const url = urljoin(config.get('domain'), datahubConfigs.owner, dataset.descriptor.name,'v',revisionId)
```

Comment:
@Mikanebu `res.flow_id` is already `{owner}/{dataset-name}/{revision-id}` so why are you splitting and after rejoining again

Reply:
Oh I see you are inserting v init

Reply:
yes, and also , `owner` is not `username`, it is `ownerid` which comes from specstore